integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -0,0 +1,85 @@
+import { getCsrfToken } from "../utils/csrf-token";
+
+let initialLongitude: string, initialLatitude: string;
+let userInputLng: string, userInputLat: string;
+
+window.addEventListener("load", () => {
+    document.getElementById("id_address")?.addEventListener("keyup", fetchCoordinates);
+    document.getElementById("id_postcode")?.addEventListener("keyup", fetchCoordinates);
+    document.getElementById("id_city")?.addEventListener("keyup", fetchCoordinates);
```

Comment:
If we perform an ajax call for each keyup event, we will spam the nominatim server with very many request.
This could cause race conditions and puts unnecessary strain on the service, so I'd suggest to either include a timeout here similar how we do it for [search suggestions](https://github.com/digitalfabrik/integreat-cms/blob/33a4ccf6fb2734fd2f89f077a045136cae88b2d6/integreat_cms/static/src/js/search-query.ts#L72-L89) or use another event, e.g. `focusout`.
```suggestion
    document.getElementById("id_address")?.addEventListener("focusout", fetchCoordinates);
    document.getElementById("id_postcode")?.addEventListener("focusout", fetchCoordinates);
    document.getElementById("id_city")?.addEventListener("focusout", fetchCoordinates);
```